app/assets/stylesheets/sephora_style_guide/variables/_icons.scss

```diff
@@ -85,3 +85,10 @@ $sephora-sprite: (
   )
 );
 // ------------------------------- Sprite icons congfig end ----------------------------
+
+// ------------------------------ SVG icons congfig start ----------------------------
+$sephora-international-countries: hong-kong singapore philippines malaysia australia new-zealand thailand indonesia
```

Comment:
Are there any rule to the ordering? If not, I think it would be better to have them sorted by alphabetical order. Same with all the files below with all the countries. It will be easier for devs to use or spot missing countries.